app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/ValueLabel/ValueLabel.style.tsx

```diff
@@ -1,5 +1,5 @@
 import styled from "styled-components";
-import { blue, mint } from "../../../../../assets/styles/colors";
+import { blue, green, mint } from "../../../../../assets/styles/colors";
```

Comment:
I see mint is unused, not connected to your changes, though.

Reply:
Done